scripts/aem.js

```diff
@@ -395,16 +395,18 @@ function decorateButtons(element) {
 
 /**
  * Add <img> for icon, prefixed with codeBasePath and optional prefix.
- * @param {span} [element] span element with icon classes
+ * @param {Element} [span] span element with icon classes
  * @param {string} [prefix] prefix to be added to icon the src
  */
 function decorateIcon(span, prefix = '') {
```

Comment:
I think we need to make this configurable. It only makes sense to add the alt attribute if we have an actual valuable description to expose via screen readers. If the icons are only used for decoration purposes, then it's usually better to just annotate them with `role="presentation" alt=" "`.

I'd rather propose something like:
```suggestion
function decorateIcon(span, prefix = '', alt = '') {
```

Reply:
I did commit this, but my only concern with this is - there's no value coming from the `decorateIcons` wrapping iteration. This function is not exported, so where should the caller source this value?

Reply:
You'd have to export the single `decorateIcon` as well and then explicitly instrument in the blocks that require proper `alt` text